boost-install.jam

```diff
@@ -288,6 +289,20 @@ rule generate-cmake-variant- ( target : sources * : properties * )
         ""
         : true ;
     }
+
+
+    local deps = [ MATCH "boost_(.*)" : [ $(ps).get <boost-install.dependency> ] ] ;
+    .info "  deps=" $(deps) ;
+
+    for local dep in $(deps) headers
+    {
+        print.text
+
+        "find_dependency(boost_$(dep) $(version) EXACT)"
```

Comment:
I wasn't sure about this because in general it will `find_dependency` more than once (for multiconfig generators.)